apps/web-giddh/src/app/services/daybook.service.ts

```diff
@@ -86,4 +86,29 @@ export class DaybookService {
         }),
             catchError((e) => this.errorHandler.HandleCatch<DayBookResponseModel, DayBookRequestModel>(e, request)));
     }
+
+    /**
+     * For Export Daybook expanded
+     * @param request 
+     * @memberof DaybookService 
+     */
+    public exportDaybookExpandedPost(request: any, branchUniqueName: string): Observable<BaseResponse<any, any>> {
```

Comment:
branchUniqueName: string

this parameter is not in you function doc, add this too

Reply:
done